### ISSUE TITLE
<math> instead of big O notation

### ISSUE THREAD

**Issue by monkoose** [ndmitchell/hoogle#357]
How to reproduce `hoogle --version 5.0.17.15`
`hoogle --json -n 1 map` the output would be
```
[
  {
    "url": "https://hackage.haskell.org/package/base/docs/Prelude.html#v:map",
    "module": {
      "url": "https://hackage.haskell.org/package/base/docs/Prelude.html",
      "name": "Prelude"
    },
    "package": {
      "url": "https://hackage.haskell.org/package/base",
      "name": "base"
    },
    "item": "map :: (a -> b) -> [a] -> [b]",
    "type": "",
    "docs": "&lt;math&gt;. <a>map</a> <tt>f xs</tt> is the list obtained by\napplying <tt>f</tt> to each element of <tt>xs</tt>, i.e.,\n\n<pre>\nmap f [x1, x2, ..., xn] == [f x1, f x2, ..., f xn]\nmap f [x1,x2, ...] == [f x1, f x2, ...]\n</pre>\n\n<pre>\n&gt;&gt;&gt; map (+1) [1, 2, 3]\n</pre>\n"
  }
]
```
As you can see value `doc` json attribute starts with some `&lt;math&gt;` but it should show big O notation `On` instead like there https://hackage.haskell.org/package/base/docs/Prelude.html#v:map

**Comment by ndmitchell** [comment]
See #353. Whoever implemented math in `haddock --hoogle` decided that `<math>` was the proper way for it to be displayed. I disagree, but that's something to raise with the Haddock team. I also disagree that its a useful feature in the base docs, but that's something to raise with the docs team. There is a hack in Hoogle HEAD that just drops the `<math>`. If that's useful to you, I can make a release?

**Comment by monkoose** [comment]
`math` doesn't bother me that much (just wanted to inform you about this issue). So it is up to you. And sorry for duplication, have searched issues with a bad pattern.

**Comment by ndmitchell** [comment]
Thanks for the report anyway - I appreciate it! Since this was fixed in HEAD, I've released 5.0.18 so people running an installed version also see the fix.
